### ISSUE TITLE
Clarification for condition table what happens if a parameter ID is provided as value

### ISSUE THREAD

**Issue by matthiaskoenig** [PEtab-dev/PEtab#541]
Hi all,
in the condition table it should be clarified what happens if a parameter ID is provided for a parameter ID value:
https://petab.readthedocs.io/en/latest/documentation_data_format.html#condition-table

`${parameterId}`
The values will override any parameter values specified in the model.

Should read:

`${parameterId}`
The values will override any parameter values specified in the model.
If a parameter ID is provided, it is interpreted as the initial parameter value.

**Comment by dweindl** [comment]
Agreed. Related to the [discussion](https://github.com/PEtab-dev/PEtab/pull/538#discussion_r859115079) in #538, where the handling of rule-targets in the condition table is clarified. Should be added there.
